sherlock/document.py

```diff
@@ -217,6 +302,26 @@ def from_dict(cls, doc: "Document", dct: Dict[str, Any]) -> "Relation":
 
 @dataclass(frozen=True)
 class Event:
+    """
+    Representation for Events within a Document.
+
+    Saves a single Event.
+    TODO: this is not consistent with Entity:
+          Entity has an abstract class (Entity) and its mentions
+          separate, whereas every Relation is counted for itself.
+
+    Attributes
+    ----------
+    doc : ``Document``
+        Document identifier the Entity belongs to
+    event_type : ``str``
+        ???
```

Comment:
see branch sherlock_version_2

Reply:
Was planning to merge sherlock_version_2 into this, thus I will keep it as such and change it later